css/index.css

```diff
@@ -104,14 +134,59 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    /* background: black; */
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
Good job allowing the parent to style the children except where the color changed, a great benefit of CSS.